Testscripts/Windows/CPU-Offline.ps1

```diff
@@ -59,11 +63,15 @@ function Main {
 				# Overwrite if storage is set
 				$isStorageWorkloadEnable = 1
 			}
+			if ($TestParam -imatch "network=yes") {
+				# Overwrite if network is set
+				$isNetworkWorkloadEnable = 1
+			}
 		}
 		Write-LogInfo "constants.sh created successfully..."
 		#endregion
 
-		if ($isStorageWorkloadEnable -eq 1) {
+		if (($isStorageWorkloadEnable -eq 1) -or (-not @("UBUNTU").contains($global:detectedDistro))) {
```

Comment:
the storage workload testing only run against ubuntu?

Reply:
No, look at this module again. This is for new disk addition. Non-Canonical VM disk layout does not have enough space in /, so git clone always failed due to insufficient disk. So I do git clone in /home/lisa/data/linux instead of /home/lisa/linux.